src/02-data-types.js

```diff
@@ -95,7 +110,13 @@ function checkIfCharacterIsInString(text, character) {}
  *  isEven(11);
  *  //> false
  */
-function isEven(num) {}
+function isEven(num) {
+  if (num % 2 == 0){
```

Comment:
Remember that comparisons evaluate to booleans, so we can just return the comparison.`num % 2 == 0`